OracleSiebel/OEL/build/multi-stage/cgw/cgw_install_update.sh

```diff
@@ -0,0 +1,24 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+#
+# Since: Dec, 2019
+# Authors: andreas.thieltges@oracle.com and duncan.ford@oracle.com
+# Description: Install Siebel Update for CGW
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+# #DF# Double check this is required.
+
+chown -R siebel /mnt/Siebel_Enterprise_Server
```

Comment:
These can be combined into a single command:
```
chown -R siebel:siebel /mnt/Siebel_Enterprise_Server
```

Reply:
Done